clients/src/main/java/org/apache/kafka/common/security/auth/KafkaPrincipal.java

```diff
@@ -43,6 +43,8 @@
  */
 public class KafkaPrincipal implements Principal {
     public static final String USER_TYPE = "User";
+    public static final String Token_Type = "Token";
```

Comment:
Just a nitpick.. All string constants should be upper case. I just want to follow the existing style guide